lib/comm_common.cpp

```diff
@@ -696,6 +698,25 @@ bool comm_gdr_blacklist() {
   return blacklist;
 }
 
+bool comm_nvshmem_enabled()
+{
+#if (defined MULTI_GPU) && (defined NVSHMEM_COMMS)
+  static bool nvshmem_enabled = true;
+#else
+  static bool nvshmem_enabled = false;
+#endif
+#ifdef MULTI_GPU
```

Comment:
shouldn't this also check for `NVSHMEM_COMMS` being defined?

Reply:
done